airbyte-integrations/connectors/source-chartmogul/source_chartmogul/chartmogul.yaml

```diff
@@ -0,0 +1,89 @@
+version: "0.2.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["entries"]
+  requester:
+    url_base: "https://api.chartmogul.com"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['api_key'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    requester:
+      $ref: "*ref(definitions.requester)"
+  customers_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      requester:
+        $ref: "*ref(definitions.requester)"
+        request_options_provider:
+          request_parameters:
+            page: "{{ (next_page_token['next_page_token'] or 0) + 1 }}"
+      paginator:
+        type: DefaultPaginator
+        url_base: "*ref(definitions.requester.url_base)"
+        pagination_strategy:
+          type: "PageIncrement"
+          page_size: 200
+        page_size_option:
+          inject_into: request_parameter
+          field_name: per_page
+    $options:
+      name: "customers"
+      primary_key: "id"
+      path: "/v1/customers"
+  activities_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      requester:
+        $ref: "*ref(definitions.requester)"
+        request_options_provider:
+          request_parameters:
+            start-date: "{% if not next_page_token %}{{ config.start_date }}{% endif %}"
```

Comment:
do we actually need to remove the start-date parameter if next_page_token exists?

Reply:
To be honest I just repeat behaviour from old implementation one to one.
I have removed this condition and checked - I got the same result.

Now `start-date: "{{ config.start_date }}"`